bin/ch_core.c

```diff
@@ -179,11 +179,20 @@ void enter_udss(struct container *c)
    if (!c->private_passwd)
       setup_passwd(c);
    // Container /tmp.
+   if ( c->tmp == NULL )
+   {
+      T_ ( host_tmp = strdup("/tmp") );
+   }
+   else
+   {
+      T_ ( host_tmp = strdup(c->tmp) );
+   }
    if (c->private_tmp) {
       tmpfs_mount("/tmp", c->newroot, NULL);
    } else {
       bind_mount("/tmp", "/tmp", BD_REQUIRED, c->newroot, 0);
```

Comment:
For your use case, would it also be good to bind-mount the host's `$TMPDIR` at container `/tmp` rather than always using the hosts's `/tmp`?